pennylane/qnode.py

```diff
@@ -999,34 +1051,37 @@ def _pd_analytic_var(self, param_values, param_idx, **kwargs):
 
         # return the variance shift rule where where_var==True,
         # otherwise return the expectation parameter shift rule
-        return np.where(where_var, pdA2-2*evA*pdA, pdA)
+        return np.where(where_var, pdA2 - 2 * evA * pdA, pdA)
 
     def to_torch(self):
         """Convert the standard PennyLane QNode into a :func:`~.TorchQNode`.
         """
         # Placing slow imports here, in case the user does not use the Torch interface
-        try: # pragma: no cover
+        try:  # pragma: no cover
             from .interfaces.torch import TorchQNode
-        except ImportError: # pragma: no cover
-            raise QuantumFunctionError("PyTorch not found. Please install "
-                                       "PyTorch to enable the TorchQNode interface.") from None
+        except ImportError:  # pragma: no cover
+            raise QuantumFunctionError(
+                "PyTorch not found. Please install " "PyTorch to enable the TorchQNode interface."
+            ) from None
 
         return TorchQNode(self)
 
     def to_tfe(self):
         """Convert the standard PennyLane QNode into a :func:`~.TFEQNode`.
         """
         # Placing slow imports here, in case the user does not use the TF interface
-        try: # pragma: no cover
+        try:  # pragma: no cover
             from .interfaces.tfe import TFEQNode
-        except ImportError: # pragma: no cover
-            raise QuantumFunctionError("TensorFlow with eager execution mode not found. Please install "
-                                       "the latest version of TensorFlow to enable the TFEQNode interface.") from None
+        except ImportError:  # pragma: no cover
+            raise QuantumFunctionError(
+                "TensorFlow with eager execution mode not found. Please install "
+                "the latest version of TensorFlow to enable the TFEQNode interface."
+            ) from None
 
         return TFEQNode(self)
 
 
-#def QNode_vjp(ans, self, params, *args, **kwargs):
+# def QNode_vjp(ans, self, params, *args, **kwargs):
 def QNode_vjp(ans, self, args, **kwargs):
     """Returns the vector Jacobian product operator for a QNode, as a function
     of the QNode evaluation for specific argnums at the specified parameter values.
```

Comment:
Just double checking, but have you made any changes to `qnode.py` apart from blacking it? The formatting changes make it hard to find.

If no other changes have been made, perhaps replace this file with the one from `master`, to avoid it showing up in the diff.

Reply:
Looks like the changes were inserted at line 371